packages/flutter_tools/test/general.shard/reporting/usage_test.dart

```diff
@@ -0,0 +1,61 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
Maybe feature_usage_test.dart, or will other usage tests go here too?

Reply:
Hopefully!

Reply:
there's a general.shard/analytics_test.dart already. Can we merge them somehow?

Reply:
d'oh! updated